File: LinguagemC++/EXERCICIO_RETANGULO.cpp
#include <bits/stdc++.h>

using namespace std;

int main ()
{
    //Calcula a área, perímetro e diagonal do retângulo
    
    double base, height, area, perimeter, diagonal;
    
    cout << "Base do retangulo: ";
    cin >> base;
    cout << "Altura do retangulo: ";
    cin >> height;
    
    area = base * height;
    
    perimeter = (base + height) * 2;
    
    diagonal = sqrt(pow(base,2) + pow(height,2));
    
    cout << fixed << setprecision(4);
    cout << "Area: " << area << endl;
    cout << "Perietro: " << perimeter << endl;
    cout << "Diagonal: " << diagonal << endl;
    
    return 0;  
}

File: LinguagemPython/exercicioRetangulo.py
# Informa a área, perímentro e diagonal do retângulo

import math

base: float
height: float
area: float
perimeter: float
diagonal: float

base = float(input("Base do retângulo: "))
height = float(input("Altura do retângulo: "))

area = base * height
perimeter = (base + height) * 2
diagonal = math.sqrt(math.pow(base, 2) + math.pow(height, 2))

print(f"Área: {area:.4f}")
print(f"Perímetro: {perimeter:.4f}")
print(f"Diagonal: {diagonal:.4f}")
